app/Http/Controllers/CategoriesController.php

```diff
@@ -0,0 +1,72 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Models\Categories;
+
+use Illuminate\Http\Request;
```

Comment:
No es necesario dar un espacio entre cada importación de librerías
```suggestion
namespace App\Http\Controllers;
use App\Models\Categories;
use Illuminate\Http\Request;
```

Reply:
Vale , no se porque se puso así pero lo arreglo